### DISCUSSION TITLE
Exec format error

### DISCUSSION THREAD

**Discussion by CyberCyclone** [prometheus/snmp_exporter#1101] [Q&A]
### Host operating system: output of `uname -a`
Linux orangepi5 5.10.160-rockchip-rk3588 #1.1.8 SMP Mon Nov 13 11:33:11 CST 2023 aarch64 aarch64 aarch64 GNU/Linux

### snmp_exporter version: output of `snmp_exporter -version`
0.25.0.darwin-arm64

### What did you expect to see?
The binary to run as expected.

### What did you see instead?
```
cgibson@orangepi5:/usr/local/bin$ ./snmp_exporter -h
-bash: ./snmp_exporter: cannot execute binary file: Exec format error
cgibson@orangepi5:/usr/local/bin$ sudo ./snmp_exporter -h
./snmp_exporter: 1: ����
                        p: not found
./snmp_exporter: 2: Syntax error: word unexpected (expecting ")")
cgibson@orangepi5:/usr/local/bin$ 
```

I'm pretty sure the OS should be compatible, so is there something I'm missing? Thanks!

**Comment by SuperQ** [comment]
What do you get for:

```
file ./snmp_exporter
```

**Reply by CyberCyclone** [reply]
```
./snmp_exporter: Mach-O 64-bit arm64 executable, flags:<|DYLDLINK|PIE>
```

**Reply by SuperQ** [reply]
That's for MacOS.
